packages/parent-bridge/contracts/parent-bridge/src/main.rs

```diff
@@ -47,6 +47,11 @@ const ANYONE_CAN_PAY_CODE_HASH: [u8; 32] = [
     230, 131, 176, 65, 57, 52, 71, 104, 52, 132, 153, 194, 62, 177, 50, 109, 90, 82, 214, 219, 0,
     108, 13, 47, 236, 224, 10, 131, 31, 54, 96, 215,
 ];
+const DEPOSIT_LOCK_CODE_HASH: [u8; 32] = [
```

Comment:
Is this the value of code hash in test nest or live? Also hard coding this prevents local development is there a way to reuse or read from .env file

Reply:
No.